x-pack/plugins/security_solution/common/detection_engine/schemas/request/perform_bulk_action_schema.ts

```diff
@@ -41,6 +40,12 @@ export enum BulkActionEditType {
   'set_schedule' = 'set_schedule',
 }
 
+export const throttleForBulkActions = t.union([
+  t.literal('rule'),
+  TimeDuration({ allowedUnits: ['h', 'd'] }),
```

Comment:
@jpdjere why do we need to introduce this limitation for throttle? While it reflects available UI options ( days, weeks, hours),  it will limit range of available values when using API calls directly

Reply:
This is a valid concern. WDYT @banderror ? Should we allow for more options via the API than via the UI?

Current allowed values via the UI are:
`1h` -> Hourly
`1d` -> Daily
`7d` -> Weekly

Reply:
I think we shouldn't allow for more options via the API than via the UI unless there are explicit requirements for that. Adding more degrees of freedom often causes more bugs and overlooked edge cases in the long run.

Reply:
> I think we shouldn't allow for more options via the API than via the UI unless there are explicit requirements for that. Adding more degrees of freedom often causes more bugs and overlooked edge cases in the long run.

In that case, looks like we need to limit values just to `1h`, `1d` and `7d`. Otherwise, you still will be able to pass something as `50d` or `2h`, which is not in UI, but still allowed through API call

Reply:
@vitaliidm great observation! While `2h` shouldn't be a big deal, something like `50d` sounds like an extreme and potentially dangerous value which I'd be fine to disallow in the schema. I'd leave it up to @jpdjere to address it right away if he has enough time or to create a ticket for us to address later.

Reply:
FYI `throttle` is documented as:

<img width="729" alt="Screenshot 2022-09-26 at 12 10 29" src="https://user-images.githubusercontent.com/7359339/192251106-084c22e1-0c53-4259-9034-1ec1190824ed.png">

Reply:
@banderror @vitaliidm 

Please take a look [at the latest commit.](https://github.com/elastic/kibana/pull/140626/commits/1a863c65348af5400d8a21db8443a30a95558a30)

I expanded `TimeDuration`'s api so that it takes either `allowedUnits` OR `allowedDurations` as options (but not both).

So we can use for Bulk Scheduling as:


And for defining the payload for `throttle` as:


Let me know what you think.